src/quo2/components/avatars/channel_avatar/style.cljs

```diff
@@ -0,0 +1,26 @@
+(ns quo2.components.avatars.channel-avatar.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def lock-icon-size 12)
+
+(defn outer-container
+  [{:keys [big? color]}]
+  {:width            (if big? 32 24)
+   :height           (if big? 32 24)
+   :border-radius    (if big? 32 24)
```

Comment:
Do you think it's worth it to have these values as variables?

Reply:
Sure, it's worth it :)